package main.modelo.excepciones;

public class respuestaVaciaException extends Exception {
	public respuestaVaciaException() {
		super("Vaya, a√∫n no ha sido respondida la pregunta");
	}
}
